s3/config/config_presign.go

```diff
@@ -0,0 +1,29 @@
+package config
+
+import (
+	"github.com/HomesNZ/env"
+	validation "github.com/go-ozzo/ozzo-validation"
+)
+
+type PresignConfig struct {
+	AccessKeyID     string
+	SecretAccessKey string
+	Region          string
+}
+
+func NewPresignConfigFromEnv() *PresignConfig {
```

Comment:
Why do we need a different config from the one we already have? Can config in https://github.com/HomesNZ/go-common/blob/master/s3/config/config.go be re-used?